packages/relayer/src/services/relay.ts

```diff
@@ -87,12 +77,44 @@ export const relayer = async (
     };
   }
 
+  if (reciver_storage) {
+    await account.functionCall({
+      contractId: tokenId,
+      methodName: "storage_deposit",
+      args: {
+        account_id: reciver_storage,
+        registration_only: true,
+      },
+      gas: AttachedGas as any,
+      attachedDeposit: "2350000000000000000000" as any,
```

Comment:
Why are we using this fixed amount for storage fees?
The way I see it we have 2 options here:
1. Use a large value, to guarantee that it is enough to cover the contract's storage cost (I'd suggest 1 NEAR). This works because `storage_deposit` calls are supposed to return excess deposits to caller
2. Use `storage_balance_bounds` view call to check the minimum deposit amount and deposit exactly that

I'd go with 1 for simplicity sake